pom.xml

```diff
@@ -194,7 +190,7 @@
     </reporting>
 
     <scm>
-      <connection>scm:git:git://github.com/${gitHubRepo}.git</connection>
+      <connection>scm:git:https://github.com/${gitHubRepo}.git</connection>
```

Comment:
Note for me: This may affect operations during releasing.
This change is also applied to empty-plugin archetypes: https://github.com/jenkinsci/archetypes/pull/367

Reply:
I think that changing `developerConnection` may affect operations during releasing, but not `connection` (as I am doing here).